app/src/main/res/layout/activity_accessibility_settings.xml

```diff
@@ -109,7 +109,7 @@
                     android:textColor="?attr/normalTextColor" />
             </LinearLayout>
 
-            <com.duckduckgo.mobile.android.ui.view.SectionDivider
+            <com.duckduckgo.mobile.android.ui.view.divider.HorizontalDivider
```

Comment:
There's a gap now but it's the same grey color as the line, should it possibly be a margin instead of padding?

![Screenshot_20221003_235821](https://user-images.githubusercontent.com/3471025/193700620-c0ba7191-eab8-4e59-86ef-b152bbe5230c.png)

Reply:
fixed